templates/App/Datum/_datum.html.twig

```diff
@@ -55,20 +55,22 @@
                 >
             </div>
         {% endif %}
-    </div>
-    <div class="col s12 l5 xl4">
-        <div class="row">
-            <div class="col s12 l5 offset-l1 xl4 offset-xl2">
-                <input
-                        type="text"
-                        id="data_{{ iteration }}_visibility"
-                        name="{{ entity }}[data][{{ iteration }}][visibility]"
-                        maxlength="255"
-                        value="{{ label }}"
-                        placeholder="public"
-                >
-                TODO ADD VISIBILITY CHOICE HERE
-            </div>
+
```

Comment:
Removed the `</div>` by mistake?
No need for extra `\n`?

Please consider

Reply:
Every opening `<div>` has a corresponding closing `</div>` (there are 8 of each), so either the mistake was made earlier, or github is displaying the diff strangely. Currently the divs are correct

Reply:
Removed the extra `\n`

Reply:
For now, I see we only have rebase and merge enabled, so please squash fixes.